src/features/currentTodo.ts

```diff
@@ -1,10 +1,15 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    get: (curretTodo: Todo | null, { payload }: PayloadAction<Todo>) => {
```

Comment:
There is a typo in the parameter name `curretTodo`. It should be `currentTodo` to match the intended state variable name.